styles/base.less

```diff
@@ -466,3 +466,16 @@ atom-text-editor.is-focused .line.cursor-line {
   font-style: italic;
   color: #ffc600;
 }
+
+/*Nested elements in JSX*/
+.JSXNested {
+  color: #F8F8F8;
+}
+
+.entity.function.js {
+  color: #CCCCCC;
+}
```

Comment:
This doesn't appear to match with the Sublime Text version's styling for functions (see screenshot). I want to keep the Atom version as close as possible to the Sublime Text version, so I'd like to leave this change out.

(Sublime on left, Atom with this PR's changes on right)
<img width="761" alt="screen shot 2017-01-10 at 22 40 53" src="https://cloud.githubusercontent.com/assets/108518/21834848/666fa1a6-d786-11e6-8dd7-ac29b0b76f6d.png">